quit/conf.py

```diff
@@ -110,7 +110,7 @@ def __initstoreconfig(self, namespace, upstream, targetdir, configfile):
         return
 
     def hasFeature(self, flags):
-        return flags == (self.features & flags)
+        return flags == (self.features and flags)
```

Comment:
This has to be an `&` because the features are set as binary flags.

Reply:
Yes, this is only related to changes for FEATURES